pkg/proc/eval.go

```diff
@@ -427,6 +427,8 @@ func (scope *EvalScope) LocalVariables(cfg LoadConfig) ([]*Variable, error) {
 }
 
 // FunctionArguments returns the name, value, and type of all current function arguments.
+// If the cfg LoadConfig parameter is nil the values of the function arguments will not
```

Comment:
I think this is a leftover change, cfg can't be nil.

Reply:
That's correct. Removed.